docs/ref/utils.txt

```diff
@@ -501,7 +501,7 @@ Atom1Feed
         z = person.friends         # does not call
         x is z                     # is True
 
-.. function:: allow_lazy(func, *resultclasses)
```

Comment:
Please keep at least a stub of documentation for `allow_lazy()` and mark it deprecated.